posenet/src/mobilenet.ts

```diff
@@ -76,6 +76,7 @@ const mobileNet50Architecture: ConvolutionDefinition[]  = [
 // clang-format on
 
 const VALID_OUTPUT_STRIDES = [8, 16, 32];
+// tslint:disable-next-line:no-any
 export function assertValidOutputStride(outputStride: any) {
```

Comment:
I wasn't sure what to do here, as I want this to accept any argument and then validate it